docs/tutorials/pyg_integration.ipynb

```diff
@@ -179,7 +177,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Since training a network with PyTorch requires defining a dataset and dataloader, we can define our custom dataset that will take the **(1)** the SMILES; **(2)** the LogD measurement and **(3)** our Molfeat transformer as input to generate the data point we need for model training."
+    "Since training a network with PyTorch requires defining a dataset and dataloader, we can define our custom dataset that will take **(1)** the SMILES, **(2)** the LogD measurement, and **(3)** our Molfeat transformer as input to generate the data point we need for model training."
```

Comment:
```suggestion
    "Since training a network with PyTorch requires defining a dataset and dataloader, we can define our custom dataset that will take **(1)** the SMILES, **(2)** the LogD measurement, and **(3)** our molfeat transformer as input to generate the data point we need for model training."
```